nemo2riva/schema.py

```diff
@@ -91,8 +90,14 @@ def get_export_config(export_obj, args):
             "Format `{}` is invalid. Please pick one of the ({})".format(conf.export_format, supported_formats)
         )
 
-    if args.cache_support is not None:
-        conf.cache_support = args.cache_support
+    # TODO: read from schema
+    conf.export_args = {}
+    if args.config:
+        kv = dict(map(lambda s: s.split('='), args.config))
+        conf.export_args.update(kv)
```

Comment:
It might be useful to do some error checking / giving useful error messages. Something like

```suggestion
        for key_value in args.config:
            lst = s.split("=")
            if len(lst) != 2:
                raise Exception("Use correct format ...")
            k, v = lst
            conf.export_args[k] = v
```

Reply:
Sure, let's do that. I will do another iteration.